app/routes/polls/$id/index.tsx

```diff
@@ -74,16 +74,29 @@ const findCurrentStreakLength = (streak: boolean[]) => {
 	return streak.length;
 };
 
+export const PENALTY_SCORE = -1;
+
+const getCorrectPointsForVotedAnswers = (
+	parsedVoted: Voted[],
+	poll: PollData
+) =>
+	parsedVoted.reduce((acc, voted) => {
+		const answer = poll.answers.find(
+			(answer) => answer.id === voted.answerId
+		);
+		return acc + (answer?.points || PENALTY_SCORE);
```

Comment:
This is the culprit: When the answer field doesn't have points, apply -1 to the points. 

Case Matthijs: 
Applied all possible answers, which should result in 10 points. Got 9. Voted one answer "wrong", which didn't have a "points" field, thus returning -1. If the points field existed.

When editing: 
Not sure why this is going wrong. Points field is updated, and assigned points, so should be 10.